sceptre/config/reader.py

```diff
@@ -330,7 +330,9 @@ def _render(self, directory_path, basename, stack_group_config):
             template = jinja_env.get_template(basename)
             self.templating_vars.update(stack_group_config)
             rendered_template = template.render(
-                self.templating_vars, environment_variable=environ
+                self.templating_vars,
+                stack_group_path=self.context.command_path.split(sep),
```

Comment:
Should we just call this `command_path`? It could be slightly misleading otherwise since the path could be a 'stack group' or a 'stack'.

Reply:
Yes, good idea. Will make the change.